client/src/components/TaskCard/TaskCard.js

```diff
@@ -24,6 +29,7 @@ const TaskCard = ({
 
   const [startDate, startTime] = getDateTime(startDateTime);
   const [endDate, endTime] = getDateTime(endDateTime);
+  const [showModal, setModal] = useState(false);
```

Comment:
I would also rename this to showFeedbackModal` and `setShowFeedbackModal`

Reply:
Renamed.